src/App.js

```diff
@@ -5,11 +5,14 @@ import ActivityCards from './components/ActivityCard/ActivityCards';
 import DurationFilter from './components/DurationFilterButton/DurationFilter';
 import TypeFilter from './components/TypeFilterButton/TypeFilter';
 import db from './lib/activityDB';
+import FavoritesPage from './pages/FavoritesPage';
 
 export default function ActivityApp() {
   const [activities, setActivities] = useState(db);
   const [typeFilterValue, setTypeFiltervalue] = useState('all');
   const [durationFilterValue, setDurationFilterValue] = useState('all');
+  const [favorite, setFavorite] = useState([]);
+  const [isHidden, setIsHidden] = useState(true);
```

Comment:
I've changed it. Tried to find something in the web why its better to start with `false` but couldn't find anything useful. Maybe because `false` is default if nothing is given or empty/undefined etc?

Reply:
Yes, think about the following case: You develop as a part of team of developers and an other developer want to use your state isHidden. He would expect that the initial state is false and and has to be setted to true.